app/models/pydantic/assets.py

```diff
@@ -19,7 +19,7 @@ class Asset(BaseRecord):
     status: AssetStatus = AssetStatus.pending
     is_managed: bool
     is_downloadable: bool
-    metadata: AssetMetadata
+    metadata: Optional[Union[AssetMetadataOut, BaseModel]]
```

Comment:
I forget, why do you need to allow Basemodel? I think I saw why elsewhere in the PR but I can't find it ATM.

Reply:
This is to allow returning empty metadata responses since we allow creating assets without associated metadata.